row_059_app.py

```
@@ -0,0 +1,609 @@
+    return JSONResponse(content=tools_list)
+
+@app.post("/api/tools")
+async def add_tool(
+    request: Request,
+    toolName: str = Form(...),
+    toolDescription: str = Form(...),
+    toolFile: UploadFile = File(...),
+    toolSensitiveHeaders: Optional[str] = Form(None),
+    toolSensitiveBody: Optional[str] = Form(None),
+# {fact rule=unrestricted-file-upload@v1.0 defects=1}
+    username: str = Depends(get_current_user)
```

Ruff (0.13.2)

Unused function argument: `username`

(ARG001)